### PR TITLE
Create SJET-AOCODAF405V3.config

### PR DESCRIPTION
**Pull request by 634296207**
Create SJET-AOCODAF405V3.config

### CODE REVIEW

**Comment by lida2003** [comment]
LGTM, please do self test. Make sure self-test is passed, and add comment below

```
F405V3 Flash V2/V3 self test passed.
```

File: configs/default/SJET-AOCODAF405V3.config [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+# feature
+feature TELEMETRY
+feature RX_SERIAL
+feature OSD
+feature GPS
+feature LED_STRIP
+
+# serial
+serial 1 64 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
```
Comment (lida2003) [inline configs/default/SJET-AOCODAF405V3.config:121]: Why there are two definitions for serial 1 "serial 1 64 115200 57600 0 115200"?

**Comment by 634296207** [comment]
F405V3 Flash V2/V3 self test passed.

**Comment by 634296207** [comment]
F405V3 Flash V2(HW) self test passed.

**Comment by lida2003** [comment]
Add default features, which should be added when PR is created.

Just copied from https://github.com/Aocoda-RC/unified-targets/pull/2/commits/f2a0bfab73daed7f8c12c9b5f2a9c24e2ff34ffc
---

1. Select GPS for serial port 1 and vtx for serial port 2
2. LED lights are turned on by default
3. The default gps protocol is ublox
4. The receiver selects crsf protocol by default
5. The default motor protocol is Dshot600
6. Add user2 mode,
7. Bluetooth and vx (9v) are enabled by default

**Pull request merged**